--- Adafruit_MotorHAT_Motors.py	2021-01-19 18:32:37.325853637 -0800
+++ Adafruit_MotorHAT_Motors.py	2021-01-19 18:31:26.492054822 -0800
@@ -172,13 +172,13 @@ class Adafruit_DCMotor:
         pwm = in1 = in2 = 0
 
         if (num == 0):
-                 pwm = 8
-                 in2 = 9
-                 in1 = 10
+                 pwm = 5
+                 in2 = 4
+                 in1 = 3
         elif (num == 1):
-                 pwm = 13
-                 in2 = 12
-                 in1 = 11
+                 pwm = 0
+                 in2 = 1
+                 in1 = 2
         elif (num == 2):
                  pwm = 2
                  in2 = 3
@@ -224,7 +224,7 @@ class Adafruit_MotorHAT:
     INTERLEAVE = 3
     MICROSTEP = 4
 
-    def __init__(self, addr = 0x60, freq = 1600, i2c=None, i2c_bus=None):
+    def __init__(self, addr = 0x40, freq = 1600, i2c=None, i2c_bus=None):
         self._frequency = freq
         self.motors = [ Adafruit_DCMotor(self, m) for m in range(4) ]
         self.steppers = [ Adafruit_StepperMotor(self, 1), Adafruit_StepperMotor(self, 2) ]
